codegen/data_test.go

```diff
@@ -41,9 +43,11 @@ func TestData_Directives(t *testing.T) {
 						},
 					},
 				},
-				Name:    "excludeDirective",
-				Args:    nil,
-				Builtin: false,
+				Name: "excludeDirective",
+				Args: nil,
+				DirectiveConfig: config.DirectiveConfig{
+					SkipRuntime: false,
```

Comment:
Used composition here in the underlying struct so I needed to update the instantiation.